app/views/memberships/edit.html.erb

```diff
@@ -1,19 +1,21 @@
 <% content_for :page_title, "Manage team member" %>
 
-<%= link_to "Back", :back, class: "govuk-back-link" %>
 <%= render "confirm_remove_team_member" if params[:remove_team_member] %>
 
-<h1 class="govuk-heading-l govuk-!-margin-top-2 govuk-!-margin-bottom-3"><%= @membership.user.name %></h1>
+<%= render "confirm_change_permissions", membership: @membership, permission_level_data: @permission_level_data if params[:confirm] %>
+
+<h1 class="govuk-heading-l govuk-!-margin-top-2 govuk-!-margin-bottom-3">Manage permissions for <%= @membership.user.name %></h1>
 <p class="govuk-body govuk-!-margin-bottom-4"> <%= @membership.user.email %> </p>
 
+<%= render "memberships/minimum_two_administrators_required" %>
+
```

Comment:
I think you can simplify the code a bit by letting the form below submit a 'get' request to the `edit_membership_path` path and add a `hidden_field` with `confirm =true` That way, the code is similar to the confirmation of removing a user and you can remove the `if-then-else` in the `update` method of the controller